autoparams/src/test/java/org/javaunit/autoparams/AutoSourceSpecs.java

```diff
@@ -253,4 +253,23 @@ void sut_correctly_creates_local_date_time_object(LocalDateTime value) {
         assertThat(value).isNotNull();
         assertThat(value).isInstanceOf(LocalDateTime.class);
     }
+
+    @ParameterizedTest
+    @AutoSource
+    void sut_correctly_creates_local_date_with_arbitrary_value(LocalDate value1, LocalDate value2) {
+        assertThat(value1).isNotEqualTo(value2);
+    }
+
+    @ParameterizedTest
+    @AutoSource
+    void sut_correctly_creates_local_time_with_arbitrary_value(LocalTime value1, LocalTime value2) {
+        assertThat(value1).isNotEqualTo(value2);
+    }
+
+    @ParameterizedTest
+    @AutoSource
+    void sut_correctly_creates_local_date_time_with_arbitrary_value(LocalDateTime value1,
+                                                                    LocalDateTime value2) {
```

Comment:
```suggestion
    void sut_correctly_creates_local_date_time_with_arbitrary_value(
        LocalDateTime value1, LocalDateTime value2) {
```

다른 코드의 포멧과 일관되게 이렇게 수정하는 것이 어떨까요?

Reply:
수정했습니다.